src/layers/domain/core/fhir_transform.py

```diff
@@ -17,13 +17,10 @@ def create_product_team_from_fhir_org_json(
         our_model=cpm_model.Organization,
     )
     org = Root.create_ods_organisation(
-        id=fhir_org.partOf.identifier.id,
+        ods_code=fhir_org.partOf.identifier.id,
         name=fhir_org.partOf.identifier.value,
     )
-    user = Root.create_user(
-        id=fhir_org.contact[0].telecom[0].value, name=fhir_org.contact[0].name.text
-    )
-    (product_team, event) = org.create_product_team(fhir_org.id, fhir_org.name, user)
+    product_team = org.create_product_team(fhir_org.id, fhir_org.name)
```

Comment:
i'd favour keyword args to ensure we don't introduce refactoring bugs given that these models are all quite volatile

```suggestion
    product_team = org.create_product_team(id=fhir_org.id, name=fhir_org.name)
```

Reply:
Yeah, I've been settling into doing that.  Much prefer it.  There's just some that are missed.